deep_qa/layers/backend/envelope.py

```diff
@@ -49,7 +49,7 @@ def compute_output_shape(self, input_shape):
     @overrides
     def call(self, inputs, mask=None):
         span_begin, span_end = inputs
-        after_span_begin = cumulative_sum(span_begin)
-        after_span_end = cumulative_sum(span_end)
+        after_span_begin = K.cumsum(span_begin)
```

Comment:
Huh, that's weird.  Looks like this addition was more recent than I thought: https://github.com/fchollet/keras/commit/31ecfb28c3202b4e551027bb1f2aef39a9eee936.  If you update `requirements.txt` to point to Keras 2.0.3, it should work.